docs/release-notes/.FSharp.Compiler.Service/10.0.200.md

```diff
@@ -0,0 +1,5 @@
+### Fixed
```

Comment:
Shouldn't it be 10.0.100?

https://github.com/dotnet/fsharp/blob/4bf3bca8f06dc92fd47350c7219d396c0c8d1075/eng/Versions.props#L15-L17

Reply:
Yes